### ISSUE TITLE
Hyperdimension Neptunia V [BLES01788] Vulkan crash regression

### ISSUE THREAD

**Issue by Silanda** [RPCS3/rpcs3#6735]
Upon starting a new game the emulation fails with the error "{RSX [0x00d9b8c]} RSX: class std::runtime_error thrown: Verification failed: 
(in file I:\GitHub\rpcs3\rpcs3\Emu\RSX\VK\VKCompute.h:175)" after the initial text part of the opening (pressing a button skips straight to the crash).

Builds 0.0.7-8616 and earlier work, so it seems likely that the regression was caused by #6448

OpenGL appears to be unaffected and still works.

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/3709415/RPCS3.log.gz)

**Comment by kd-11** [comment]
That regression was fixed by two followup PRs, with the last one bring https://github.com/RPCS3/rpcs3/pull/6498

**Comment by Silanda** [comment]
Yes that seems to be the case. It did start working again after #6498, but it doesn't work in the latest builds. I'm in desperate need of sleep at the moment so I'll test tomorrow to see where it stopped working again.

**Comment by Silanda** [comment]
Hmm. I've done some more testing and contrary to my previous post I'm still getting the problem with #6498 (Build v0.0.7-8666-d1603fbb), sometimes with a fatal error and program crash. 

The problem appears to be intermittent. After further testing I've found that the game does sometimes manage to pass the point it usually crashes. This is actually true of the latest build as of this time as well (v0.0.7-8852-e9ea226e). It doesn't always crash as I previously thought.

I have tried deleting the game's cache directory between runs to eliminate that as a possible source of the problem.

Here's a log from v0.0.7-8666 where a fatal error occurred, though I don't seem to be seeing the fatal errors in very recent builds. 

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/3711004/RPCS3.log)

**Comment by kd-11** [comment]
Needs an update, things have changed since then with regards to texture decoding.

**Comment by Silanda** [comment]
There could be an issue my end, but now there seems to be a problem that prevents me from testing this under the same conditions as before: in both Victory and Mk2 I'm now getting no display output unless I enable "Force CPU Blit Emulation".

I probably should have mentioned this before, but IIRC that option already stopped the crash from happening. When I get the chance I'll try to track down when the game stopped displaying graphics without it.

**Comment by Silanda** [comment]
Black screen problem seems to be gone with pr #7330, and I'm not experiencing the crash.

**Comment by BuIlDaLiBlE** [comment]
Black screen issues seem to relate to DLC not being able to be decrypted, as it spammed tons of "invalid hash" and errors of "decrypting data" in the log. However, aside from that there are still some minor graphical glitches. They seem to appear after some time, not at the beginning of the game. These glitches include corruption of controller buttons and character pictures. I took two RSX captures, the first one is when graphics are ok, the second one is when they have become corrupted. I'm using US version of the game though but I don't believe that matters.
[BLUS31150 Regression RSX.7z](https://bpnet.host/files/BLUS31150%20Regression%20RSX.7z)

**Comment by kd-11** [comment]
Closing as the crash reported in this bug tracker is fixed.

@BuIlDaLiBlE You may open a separate issue unless the bugs are only present with PR https://github.com/RPCS3/rpcs3/pull/7330 in which case it makes more sense to report to https://github.com/RPCS3/rpcs3/issues/7240 which already tracks these issues anyway. Remember to attach save files for testers to reproduce the problem there too.
